frontend/ui/elements/screen_eink_opt_menu_table.lua

```diff
@@ -13,14 +13,15 @@ local eink_settings_table = {
                 G_reader_settings:saveSetting("low_pan_rate", Screen.low_pan_rate)
             end,
         },
+        require("ui/elements/flash_ui"),
+        require("ui/elements/flash_keyboard"),
```

Comment:
I think these are not that eInk specific. But OK if you need to make room, as it's most probably only on eInk that one will want to disable them (even if I disabled them on my Android phone).

Reply:
Yeah, I mainly wanted it to be near the "avoid mandatory black flashes" one, even if they're not *technically* eInk-specific ;).

Reply:
The fact that the eInk submenu is never hidden anywhere cinched the deal ;).